spec/index.bs

```diff
@@ -8013,7 +8013,11 @@ Copies between depth textures can only happen within the following sets of forma
   - {{GPUTextureFormat/depth16unorm}}, {{GPUTextureFormat/r16uint}}
   - {{GPUTextureFormat/depth24plus}}, {{GPUTextureFormat/depth24plus-stencil8}} (depth aspect)
 
-The {{GPUTextureFormat/depth32float}} format can only the destination if copied from another texture of the same format. It can also be a source of copies into {{GPUTextureFormat/r32float}}.
+Additionally, {{GPUTextureFormat/depth32float}} textures can be copied into {{GPUTextureFormat/depth32float}} and {{GPUTextureFormat/r32float}} textures.
```

Comment:
This says `depth32float` can be copied into `depth32float` which doesn't seem super helpful to have in addition to other rules. Did we mean `depth32float-stencil8`?

Reply:
No, this was my intent. Are you saying we don't need to specify it because copies within the same format are always valid? I'd argue the list just above this one also specifies copies within the same format are valid, but isn't as explicit.

The point is that nothing can copy into depth32float except for depth32float.

I forgot about depth32float-stencil8, it's not even mentioned anywhere in this section yet, but it needs to be.